### PR TITLE
Update docs re sign-up with premium subscription and sync db data

### LINKED ISSUE #1554: Improve user guide on how to sync the DB properly for premium users between machines.
## Description

I am opening an issue for that to keep track of that.

@kelsos Can you add a note to improve our user guide for the above? ^ I am not sure if it's properly documented how to sync the DB properly for premium users between machines.

_Originally posted by @LefterisJP in https://github.com/rotki/rotki/issues/1542#issuecomment-703323816_

### PR DESCRIPTION
**Pull request by vnavascues**
Updated user guides sign-up/sign-in sections as required in #1554 and to address #1542.
Extended troubleshooting with common premium subscription problems re sing-up and unsynced data.
Also removed the section re add custom ethereum tokens.

Closes #1554

### CODE REVIEW

File: docs/usage_guide.rst [hunk 1]
```diff
@@ -9,29 +9,53 @@ Introduction
...
+  - **Password**: :red:`Do not forget this password`. It is used to encrypt all your local files. For premium subscription, it also allows to save your database in the Rotki Server and sync it in multiple devices.
+
+If you have purchased a premium subscription you can also add the **API Key** and the **secret** here. See the section `Sync data with Rotki Server <sync_data_with_rotki_server_>`_ to know how to sync your data with Roki Server (this option is disabled by default).
 
 .. image:: images/sc_newacc2.png
    :alt: Creating a new account with a premium Rotki API key/secret pair
    :align: center
 
-If you have purchased a premium subscription you can also add the API Key and the secret here. This will essentially populate the new account with the data saved in the server for your premium subscription. Important thing to note here is that the password of the new account must be the same as the data saved in the server. If not, opening the database will fail.
+All accounts are created in the rotki directory which is located in: ``$HOME/.rotkehlchen``. Each user has their own subdirectory.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:35]: Hmm this is wrong and outdated. Can you change it? The location of the rotki directory depends on the OS. The locations are in the documentation here: https://rotki.readthedocs.io/en/latest/usage_guide.html?highlight=Library#rotki-data-directory

File: docs/usage_guide.rst [hunk 2]
```diff
@@ -9,29 +9,53 @@ Introduction
...
+
+
+Sign-up restoring database (premium subscription)
```
Comment (LefterisJP) [inline docs/usage_guide.rst:38]: Sign-up restoring database (premium subscription) 

->

Create a new account that restores a backed up database (premium user only)

File: docs/usage_guide.rst [hunk 3]
```diff
@@ -9,29 +9,53 @@ Introduction
...
+===================================================
+
+If you have a premium subscription, your database is already backed up in the Rotki Server and you want to sign-up restoring it (same or another device), you must add the **API key/secret** and :red:`use the same password`. If the password is not the same, opening the database will fail.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:41]: > your database is already backed up in the Rotki Server

No that's not true. They have to enable this option. They may not have a backup.

Comment (LefterisJP) [reply docs/usage_guide.rst:41]: > sign-up restoring it (same or another device)

Don't use the word sign-up. This is a very confusing lingo for a local application. Say: "Create a new local account that restores the backed up database, for example in a different device, you must ....

File: docs/usage_guide.rst [hunk 4]
```diff
@@ -44,6 +68,24 @@ If after you have set up premium you wish to replace or disassociate keys with t
...
+Sync data with Rotki Server
+-----------------------------
+
+To back up your data in the Rotki Server switch on "Allow data sync with Rotki Server". This allows you to restore it later on in any account/device as long as :red:`the same API Key/secret and account password are used`.
+
+.. image:: images/rotki_premium_set_sync_data.png
+   :alt: Sync data with Rotki Server
+   :align: center
+
+Bear in mind that in case of using multiple accounts/devices with the data sync enabled, the one with the most recent login will upload the most up-to-date data to Rotki Server. After that, log into another account/device may ask you to take action regarding whether replacing or not your local database with the remote one.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:82]: > After that, log into another account/device may ask you to take action regarding whether replacing or not your local database with the remote one.

After that using the same account from another device, may ask you whether you want to replace your local database with the remote one.

File: docs/usage_guide.rst [hunk 5]
```diff
@@ -529,3 +555,13 @@ Local system clock is not synchronized
...
+
+Sign-up restoring database fails
+==================================
+
+Please, make sure you are using your premium subscription API keys/secret and the same password.
+
+Data with multiple accounts/devices is not synced
+===================================================
+
+Please, make sure all your accounts have the "Allow data sync with Rotki Server" switched on, and that on each log in you take care of managing your local database as wanted (e.g. restoring you local database with the remote one). See the section `Sync data with Rotki Server <sync_data_with_rotki_server_>`_ for more information about how to sync data with multiple accounts/devices.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:567]: > you take care of managing your local database as wanted

I think I know what you are trying to say here but had to read it 3 times to understand. Can you maybe rewrite more ilke?

on each log in you make the appropriate choice when prompted to replace the local database.

File: docs/usage_guide.rst [hunk 6]
```diff
@@ -529,3 +555,13 @@ Local system clock is not synchronized
 Some remote servers (e.g. exchanges) require your local system clock synchronized with theirs. In case of not having it synchronized the request will fail and Rotki will either display a specific error message (i.e. 409 status code and a *local system clock is not sync* message) or the generic 500 error one (please, :ref:`report it to us <bug_reporting>`).
 
 Follow your OS official guidelines about how to synchronize the clock with an Internet Time Server and try again.
+
+Sign-up restoring database fails
```
Comment (LefterisJP) [inline docs/usage_guide.rst:559]: Restoring database at new account creation fails

File: docs/usage_guide.rst (hunk 3 above)
Comment (vnavascues) [reply docs/usage_guide.rst:41]: > > your database is already backed up in the Rotki Server
> 
> No that's not true. They have to enable this option. They may not have a backup.

I know it is not true. I've stressed a lot around you must switch on the sync option. I meant "A, and B and C" (but omitted the 2nd and). I will re-write it as suggested.

**Review by LefterisJP — APPROVED** [review]
LGTM has a small typo which I will just quickly fix and merge

**Pull request merged**